src/datepicker/datepicker-navigation.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+import {NavigationEvent} from './datepicker-view-model';
+
+@Component({
+  selector: 'tbody[ngb-datepicker-navigation]',
```

Comment:
Change to `ngbDatepickerNavigation`. Do we care that it is on `tbody`? Maybe we should relax this constraint so we leave open doors to supporting different markup?